README.md

```diff
@@ -31,21 +31,23 @@ jobs:
       with:
         creds: ${{ secrets.AZURE_CREDENTIALS }}
 
-    - name: 'Sync NGINX configuration to NGINX on Azure instance'
+    - name: 'Sync the NGINX configuration from the Git repository to the NGINX for Azure deployment'
       uses: nginxinc/nginx-for-azure-deploy-action@v1
       with:
         subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
         resource-group-name: ${{ secrets.AZURE_RESOURCE_GROUP_NAME }}
         nginx-deployment-name: ${{ secrets.NGINX_DEPLOYMENT_NAME }}
-        nginx-config-relative-file-path: ${{ secrets.NGINX_CONFIG_FILE }}
+        nginx-config-directory-path: config
```

Comment:
Just brainstorming here - feel free to ignore.

Not sure which usage would be more common - relative or absolute. Would it be useful to also retain `nginx-config-relative-file-path` example usage, but perhaps commented out? Perhaps something along the lines of:
```
...
# If your NGINX configs have relative path references, use something like this:
# nginx-config-relative-file-path: ${{ secrets.NGINX_CONFIG_FILE }}
```